optimade/validator/validator.py

```diff
@@ -164,27 +164,30 @@ def validate_implementation(self):
 
         # Test entire implementation
         print(f"Testing entire implementation at {self.base_url}...")
-        self._log.debug("Testing base info endpoint of %s", "info")
+        self._log.debug("Testing base info endpoint of %s", CONF.info_endpoint)
 
-        # Save base info to construct new queries
+        # Get and validate base info to find endpoints
         info_endp = CONF.info_endpoint
```

Comment:
You might even want to define this earlier to be able to use in the debug log line above?